src/Aspire.Hosting.Kubernetes/KubernetesResource.cs

```diff
@@ -159,9 +193,16 @@ private void ProcessEndpoints()
 
         foreach (var endpoint in endpoints)
         {
-            if (resource is ProjectResource && endpoint.TargetPort is null)
+            if (endpoint.TargetPort is null)
             {
-                GenerateDefaultProjectEndpointMapping(endpoint);
+                const string defaultPort = "8080";
+
+                var paramName = $"port_{endpoint.Name}".ToHelmValuesSectionName();
+
+                var helmExpression = paramName.ToHelmParameterExpression(resource.Name);
+                Parameters[paramName] = new(helmExpression, defaultPort);
+
+                EndpointMappings[endpoint.Name] = new(endpoint.UriScheme, resource.Name, helmExpression, endpoint.Name, helmExpression);
```

Comment:
Delete this, completely and match the endpoint processing logic from https://github.com/dotnet/aspire/blob/32e33a3f7edd74551132a333a533bbc9287a204a/src/Aspire.Hosting.Docker/DockerComposeEnvironmentContext.cs#L39-L83